src/pages/_middleware.ts

```diff
@@ -70,7 +73,7 @@ export function middleware(req: NextRequest, ev: NextFetchEvent) {
 	// Handle Opt-in cookies
 	const optInPlatform = params.get('optInFrom') as OptInPlatformOption
 	const hasOptedIn = Boolean(req.cookies[`${optInPlatform}-beta-opt-in`])
-	const response = NextResponse.next()
+	response = NextResponse.next()
```

Comment:
This doesn't seem to work as I was expecting. @BRKalow Curious how to use `setGeoCookie` with `NextResponse.next()` usage. Will need to use the same setup on Learn https://github.com/hashicorp/learn/pull/4504/files#diff-db97c6a1b4af3a37ea8a8a1786dc326bb1e26cdd70b92225480081e61be5b552R83

Reply:
This line creates a new response, in order for the cookie util to work we need to re-use the response that's declared from `setGeoCookie(req)`. We should be able to remove this line.

Reply:
Got it, so `NextResponse.next()` is not needed?

Reply:
Correct 👍